_includes/components/search-results/SearchResult/partials/normaliseServerResponse.js

```diff
@@ -0,0 +1,43 @@
+import { find } from 'lodash';
+import extractSnippet from './extractSnippet.js';
+
+
+const normaliseDepartmentItem = (item) => {
+  const { extras, province, financial_year: financialYear, organization = {}, title: rawTitle } = item;
+
+  const getExtrasValue = (key) => {
+    const obj = find(extras, extra => extra.key === key) || { value: null };
+    return obj.value;
+  };
+
+  const isOfficial = organization.title === 'National Treasury';
```

Comment:
I'd suggest you use the slug for this as we do elsewhere  - I think that field is called id or name.

Reply:
Cool. Will change to `orgnaization.name`.